frontend/public/components/conditions.tsx

```diff
@@ -11,15 +10,17 @@ export const Conditions: React.SFC<ConditionsProps> = ({obj}) => {
       {condition.status}
     </div>
     <div className="col-xs-3 col-sm-3">
-      {condition.reason}
+      {condition.reason ? condition.reason : '-'}
     </div>
-    <div className="col-xs-3 col-sm-5">
-      {condition.message}
+    <div className="col-xs-3 col-sm-5 co-pre-line-wrap">
+      {/* remove initial newline which appears in route messages */}
```

Comment:
Are you sure the newline isn't because you have a newline after the div? Can you try...

```jsx
<div className="col-xs-3 col-sm-5 co-pre-line-wrap">{condition.message || '-'}</div>
```

Reply:
ok, I'll try, but I'm seeing it in the YAML as well:
![image](https://user-images.githubusercontent.com/12733153/41662289-c5fa7df4-746e-11e8-93d6-fb7adbc20cee.png)

Reply:
Hi, tried 
```jsx
<div className="col-xs-3 col-sm-5 co-pre-wrap">{condition.message || '-'}</div>
```
Still seeing an initial newline

Reply:
I think _.trim with pre-line should be enough to format this without the regex